draft-ietf-tokbind-https-08.xml

```diff
@@ -222,7 +222,7 @@
 
     <abstract>
       <t>This document describes a collection of mechanisms that allow
-      HTTP servers to cryptographically bind authentication tokens
+      HTTP servers to cryptographically bind authentication or authorization tokens
```

Comment:
That would be fine with me too. Just looking to not have wording that suggests oauth == authn.